apps/core/lib/core/snitch/variant.ex

```diff
@@ -8,21 +8,20 @@ defmodule Core.Snitch.Variant do
   @type t :: %__MODULE__{}
   schema "snitch_variants" do
```

Comment:
Why are we adding `snitch` prefix

Reply:
@SagarKarwande `aise hi 😎`   cc @aviabird/all @arjun289